ansible/group_vars/telemetry.yml

```diff
@@ -13,17 +13,16 @@
 # limitations under the License.
 
 ---
-- name: uninstall prometheus
-  shell: "{{ item }}"
-  with_items:
-    - systemctl stop prometheus
-    - rm -rf /etc/systemd/system/prometheus.service
-    - rm -rf /etc/prometheus/prometheus.yml
-    - rm -rf /etc/prometheus
-    - rm -rf /usr/local/bin/promtool
-    - rm -rf /usr/local/bin/prometheus
-    - rm -rf /var/lib/prometheus
-    - userdel -r prometheus
-    - userdel -r node_exporter
-  ignore_errors: yes
-  become: yes
+# Dummy variable to avoid error because ansible does not recognize the
+# file as a good configuration file when no variable in it.
+dummy:
+
+###########
+# GENERAL #
+###########
+
+# Do you need to install telemetry tools?
+install_telemetry_tools: false
```

Comment:
Can u add `sed -i 's/^install_telemetry_tools: .*/install_telemetry_tools: '"true"'/g' ansible/group_vars/telemetry.yml` in [ci_check.sh](https://github.com/opensds/opensds-installer/blob/master/ci_check.sh) so that telemetry tool can be tested in ci environment?